ambari-agent/conf/unix/ambari-agent.service

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific
+
+# This service unit file is tuned for SLES 12.x. It is not suitable for RHEL like distributions due to different
+# paths for ExecStart/ExecStop.
+
+[Unit]
+Description=Ambari Agent Service for SLES
+Before=runlevel2.target
```

Comment:
runlevel 2 - is a system start without network
runlevel 4 - is not defined 

Why we require agent start on the stages when agent will not actually work anyways?

Reply:
Thanks for spotting it. Basically I took the Centos generated service file as start, hence it is there. I will remove.